project-reports/2024/2024-Q4-Hyperledger-Identus.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+title: 2024 Q3 Hyperledger Identus
```

Comment:
```suggestion
title: 2024 Q4 Hyperledger Identus
```

With Q4 reports, we are moving to the new TAC governance site. Could you please close this PR and re-open a PR in the https://github.com/lf-decentralized-trust/governance report and add the report to the `tac/project-updates` folder?

Reply:
I've created this PR  https://github.com/LF-Decentralized-Trust/governance/pull/44/files